taxonium_component/src/Taxonium.tsx

```diff
@@ -216,7 +216,8 @@ function Taxonium({
 
   return (
     <GlobalErrorBoundary>
-      <div className="w-full h-full flex">
+      <div className="taxonium w-full h-full flex">
```

Comment:
I'm seeing an issue here where I think the `w-full h-full` (and presumably `flex`) are having no effect as I guess they would need to be children of `.taxonium`? This means that in the preview of the taxonium website there is truncation. 

<img width="1216" height="745" alt="Image" src="https://github.com/user-attachments/assets/80c56e2d-c53d-4023-93d3-c9f48d6e900c" /> I'd be fine with just writing these out as `style={}` things to resolve that

Reply:
(then hit SARS-CoV2 for example)